servicemanager/servicemanager.go

```diff
@@ -83,6 +85,7 @@ type ServiceManager struct {
 	layerProvider       LayerProvider
 	serviceAllocator    spaceallocator.Allocator
 	downloadAllocator   spaceallocator.Allocator
+	ttlStopChannel      chan bool
```

Comment:
`validateTTLStopChannel`?
Use empty struct for such channels: `struct{}`.

Reply:
Done